### ISSUE TITLE
Setting boundary effects for erosion and dilation

### ISSUE THREAD

**Issue by jakirkham** [luispedro/mahotas#70]
It appears the convention for regions beyond the boundary to be zero when working with erosion and dilation. However, I often like to chose a different constant, namely one on binary images, so as not to completely lose objects on the borders with common structuring elements. Is there some way to do this with `mahotas`? I didn't see anything obvious from the documentation, but it doesn't mean I didn't missing something.

**Comment by luispedro** [comment]
Yes, this is hard coded for erosion/dilation, so the code would need to be extended to accept the boundary options. I'd welcome a patch, but it's not clear when I'd have time to do it myself.
